editor/editor_help.cpp

```diff
@@ -3410,26 +3410,21 @@ EditorHelpBit::HelpData EditorHelpBit::_get_theme_item_help_data(const StringNam
 	bool found = false;
 	const DocTools *dd = EditorHelp::get_doc_data();
 	HashMap<String, DocData::ClassDoc>::ConstIterator E = dd->class_list.find(p_class_name);
+
 	while (E) {
 		// Non-native theme items shouldn't be cached, nor translated.
 		const bool is_native = !E->value.is_script_doc;
 
 		for (const DocData::ThemeItemDoc &theme_item : E->value.theme_properties) {
 			HelpData current;
 			current.description = HANDLE_DOC(theme_item.description);
+
 			if (theme_item.is_deprecated) {
-				if (theme_item.deprecated_message.is_empty()) {
-					current.deprecated_message = TTR("This theme property may be changed or removed in future versions.");
-				} else {
-					current.deprecated_message = HANDLE_DOC(theme_item.deprecated_message);
-				}
+				current.deprecated_message = theme_item.deprecated_message.is_empty() ? TTR("This theme property may be changed or removed in future versions.") : HANDLE_DOC(theme_item.deprecated_message);
```

Comment:
Is there a tangible benefit to changing these to single line ternary operators instead? This puts these lines at over 193 characters long, which means a lot of folks will have to scroll to see it all.